mixins/code_editor.go

```diff
@@ -240,16 +242,26 @@ func (e *CodeEditor) KeyStroke(ev gxui.KeyStrokeEvent) (consume bool) {
 // mixins.TextBox overrides
 func (e *CodeEditor) CreateLine(theme gxui.Theme, index int) (TextBoxLine, gxui.Control) {
 	lineNumber := theme.CreateLabel()
-	lineNumber.SetText(fmt.Sprintf("%.4d", index+1)) // Displayed lines start at 1
+	lineNumber.SetText(fmt.Sprintf("%4d", index+1)) // Displayed lines start at 1
```

Comment:
Did you intend to change this from `%.4d` to `%4d`?

Reply:
I just realized, the old code is bugged.  In fact, [`vidar` overrides this](https://github.com/nelsam/vidar/blob/master/editor/editor.go#L339-L353) to get its line numbers.

lgtm.